cstar/base/base_model.py

```diff
@@ -294,3 +290,10 @@ def handle_config_status(self) -> None:
     @abstractmethod
     def get(self, target: str | Path) -> None:
```

Comment:
This is not an urgent nor a particularly helpful comment at this time, but I think a less overloaded term for this method could be `fetch`, because `get` already has nother widely-used meaning in python.

Reply:
Agree, I have been pondering a better alternative for a minute - even ignoring the conflict with python's use of get, ours is used in contexts going far beyond fetching, like `BaseModel.get()` which would be much better described as `BaseModel.setup()`. Fetch is fine, but in certain situations `get()` also creates symlinks, which I wouldn't really describe as "getting" or "fetching".

Reply:
hmm, okay, let's punt on this. I'll raise your comment as an issue.